mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -328,6 +330,38 @@ class WidgetAdapter(
         }
     }
 
+    abstract class HeavyDataViewHolder internal constructor(
+        inflater: LayoutInflater,
+        parent: ViewGroup,
+        @LayoutRes layoutResId: Int
+    ) : ViewHolder(inflater, parent, layoutResId) {
+        protected val widgetContentView: View = itemView.findViewById(R.id.widget_content)
```

Comment:
I must use `View` here, but this is generic to be used by the child view holders.